doc/explanation.rst

```diff
@@ -0,0 +1,47 @@
+Explanations
+============
+
+This section of the documentation exists to illuminate how ``autoreject`` works.
+The primary source for understanding should be the original publication [1]_,
+however the sections in this guide can make the content of that primary source
+more graspable.
+
+
+Intuition on how the - *autoreject global* - algorithm works
+------------------------------------------------------------
+- Given some MEEG data :math:`X` with the dimensions
+  :math:`trials(=epochs) \times sensors \times timepoints`
+- We want to find a threshold :math:`\phi` in :math:`\mu V` that will reject
+  noisy epochs and retain clean epochs
+- Do the following for a set of possible candidate thresholds: :math:`\Phi`
+- For each :math:`\phi_i \in \Phi` :
+    - Split your data :math:`X` into :math:`K` folds (=:math:`K` equal parts)
+      along the trial dimension
+        - Each of the :math:`K` parts will be a "test" set once, while the
+          remaining :math:`K-1` parts will be combined to be the corresponding
+           "train" set (see `k-fold crossvalidation <https://en.wikipedia.org/wiki/Cross-validation_(statistics)#k-fold_cross-validation>`_)
```

Comment:
this part is not rendered correctly

Reply:
Should be fixed through the help of [rst docs](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#bullet-lists)